zulipterminal/ui_tools/boxes.py

```diff
@@ -500,6 +501,16 @@ def footer_notify_invalid_recipient(self, error_msg: str) -> None:
         ]
         self.view.set_footer_text(footer_message, 3)
 
+    def loosing_focus_from_stream_box(self, key: str) -> bool:
+        stream_header = self.header_write_box[self.FOCUS_HEADER_BOX_STREAM]
+        stream_name_len = len(stream_header.edit_text)
+        cursor_pos = stream_header.edit_pos
+        if (is_command_key('CYCLE_COMPOSE_FOCUS', key)
+                or key in ['right', 'down']):
```

Comment:
As these command keys are configurable from `keys.py`, we shouldn't hardcode them here.

Reply:
Actually, the command that binds these keys(`right` and `down`) offers a secondary letter-key(i.e. `i` and `j` respectively). Hence, using the commands instead of keys would prevent the letter keys from getting typed in the write-box.

Reply:
I understand why you've used them. However, you can see that if the keys change from `keys.py`, it is not going to where. For example, we could have another method for figuring out whether the command keys have letters and take it further from there. In any case, hardcoding shouldn't be our last resort.